envs/monkey_zoo/blackbox/config_templates/powershell.py

```diff
@@ -23,7 +23,7 @@ class PowerShell(ConfigTemplate):
             "basic.credentials.exploit_user_list": ["m0nk3y", "m0nk3y-user"],
             "internal.classes.finger_classes": [],
             "internal.network.tcp_scanner.HTTP_PORTS": [],
-            "internal.network.tcp_scanner.tcp_target_ports": [],
+            "internal.network.tcp_scanner.tcp_target_ports": [5985, 5986],
```

Comment:
Already done in cb51394439928b8077ae46b6559406ce358a3813, rebase

Reply:
Since the change is identical, it will work itself out in the merge.